### ISSUE TITLE
I can't push suddely

### ISSUE THREAD

**Issue by zioworkscom** [gitblit/gitblit#989]
It was fine until yesterday. But I can't push anymore with these errors on SourceTree.
Some private infos blinded.

-----------------------------------------------------------------------------------------------
git -c diff.mnemonicprefix=false -c core.quotepath=false push -v --tags 14.myGroup.something master:master
Pushing to ssh://distilled@xx.xx.xx.xx:xxxx/myGroup/suba/someProject.git

fatal: Cannot receive pack: 

remote: Resolving deltas:   7% (1/14)   [K
remote: Resolving deltas:  14% (2/14)   [K
remote: Resolving deltas:  21% (3/14)   [K
remote: Resolving deltas:  28% (4/14)   [K
remote: Resolving deltas:  35% (5/14)   [K

remote: Resolving deltas:  42% (6/14)   [K
error: unpack failed: error Missing delta base 45a0ac4fe3cc3806d79dba8d56322fb281979cbc
To ssh://distilled@xx.xx.xx.xx:xxxx/myGroup/suba/someProject.git
 ! [remote rejected] master -> master (n/a (unpacker error))
error: failed to push some refs to 'ssh://myid@xx.xx.xx.xx:xxxx/myGroup/suba/someProject.git'

**Comment by gitblit** [comment]
I think you'll find the solution [here](http://gitblit.com/faq.html#H2) and the previous (imported from GoogleCode) issue #704.

    git.checkReferencedObjectsAreReachable = false

TL;DR this is a bug in JGit.  It was briefly fixed upstream but the fix was reverted because it impacted performance so in the current JGit releases this is still an issue.  It seems to be mostly triggered by rebasing/amending.  If you configure the setting mentioned in the FAQ I think you'll be able to push.

**Comment by zioworkscom** [comment]
Thanks.

git.checkReferencedObjectsAreReachable = false

 It's works.
